cmd/skopeo/inspect.go

```diff
@@ -96,7 +98,7 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		}
 	}()
 
-	rawManifest, _, err := src.GetManifest(ctx, nil)
+	rawManifest, err := retryGetManifest(ctx, nil, opts.image.maxRetry, opts.image.retryDelay, src)
```

Comment:
It is admittedly surprising, but the part that actually fetches the manifest (and thus can fail) is actually the `NewImageSource` in `parseImageSource`; this `GetManifest` is just a local data copy nowadays. I suppose the code should wrap both together.

Also, the `ConfigBlob`/`OCIConfig` calls should probably be retried, though, admittedly, that can come separately.

Reply:
More precisely, for `docker://` _as it is implemented right now_ the fetch happens in `NewImageSource`; other transports do implement this differently.